### DISCUSSION TITLE
Back button

### DISCUSSION THREAD

**Discussion by Matini97** [Budibase/budibase#2342] [Help] [answered]
I'm a beginner/currently learning SQL programing and I have a database with multiple tables, and I made a simple button setup to link from i.e., /technology/new/row to /goals/new/row. So far so good, but I have no idea of how to implement the "back" button, it should link from /goals/new/row to /technology/:id with the id of the last inputted row.
Since this Database runs on an internal Server it doesn't necessarily require going back to your last input, just the last input row of the table would be fine.
Would appreciate any ideas.
TIA

**Comment by joebudi** [comment, marked as answer]
Hey @Matini97 

Out of curiosity, what are you trying to do from a higher-level perspective? Are you attempted to build a stepped form/wizard?

**Reply by Matini97** [reply]
Hi @joebudi 
Pretty much yes.

**Reply by kevmodrome** [reply]
Dynamic forms and form wizard functionality is coming in the August release (later this month), it's probably a good idea to hold off on that part of your application for now. It would be pretty difficult to implement this at the moment unfortunately

**Reply by Matini97** [reply]
@kevmodrome 
ok good to know.
thank you and looking forward to it
